config/settings_schema.json

```diff
@@ -474,10 +474,6 @@
   {
     "name": "t:settings_schema.cards.name",
     "settings": [
-      {
```

Comment:
Removing the header because we don't need it anymore.

Reply:
Same here.

Reply:
Taking track of this here 🙏 https://github.com/Shopify/dawn/issues/986#issuecomment-997008098